### ISSUE TITLE
Customize displayed work item

### ISSUE THREAD

**Issue by StefanOverHaevgRZ** [microsoft/azure-boards-estimate#23]
Currently for Product Backlog Items, only the description is shown.
However, most of our estimates depend on the information in the acceptance criteria.

It would be nice to configure the parts that are shown during the estimation process.

**Comment by rvdp** [comment]
Indeed, my team is experiencing the same issue. We can't poker a story without agreeing on the acceptance criteria. Would much appreciate support for this.

**Comment by maikefischer** [comment]
Same!

**Comment by hangy** [comment]
This is also an issue when we want to estimate bugs, because they don't have a description field.

**Comment by spoelly** [comment]
I would really like to have this feature to!

**Comment by MOlausson** [comment]
Good feedback, added to the backlog.

**Comment by cluen** [comment]
We also need the display of acceptance criteria. 
It would be nice to see the tags of a work item as well.

**Comment by AminTi** [comment]
resolved
